includes/views/object-rubrics-settings-template.php

```diff
@@ -7,55 +7,33 @@
 	 */
 
 	$rubrics = \UBC\CTLT\WPVote\WP_Vote_Settings::get_rubrics_options();
+	do_action( 'ubc_wp_vote_setting_metabox' );
 ?>
 
-<section style="margin: 1rem 0;">
+<section class="ubc-wp-vote-metabox">
 	<label for="">
-		Override default <?php echo esc_html( $object_title ); ?> rubrics?
-		<input
-			type="checkbox"
-			name="<?php echo esc_attr( $option_name ) . 'override'; ?>"
-			onClick="onUBCWPVoteOverride( this )"
-			id="<?php echo esc_attr( $option_name ) . 'override'; ?>"
-			for="<?php echo esc_attr( $option_name ) . 'rubrics'; ?>"
-			<?php if ( get_post_meta( intval( get_the_ID() ), esc_attr( $option_name ) . 'override', true ) ) : ?>
-			checked
-			<?php endif; ?>
-		/>
+		Rubrics for <?php echo esc_html( $object_title ); ?>?
 	</label>
 
 		<ul
 			id="<?php echo esc_attr( $option_name ) . 'rubrics'; ?>"
-			<?php if ( ! get_post_meta( intval( get_the_ID() ), esc_attr( $option_name ) . 'override', true ) ) : ?>
-			class="hidden"
-			<?php endif; ?>
 		>
 		<?php foreach ( $rubrics as $key => $rubric ) : ?>
 			<li>
 				<label>
-					<input
-						type="checkbox"
-						name="<?php echo esc_attr( $option_name ) . 'rubrics[]'; ?>"
-						value="<?php echo esc_attr( $rubric->name ); ?>"
-						<?php if ( is_array( get_post_meta( intval( get_the_ID() ), esc_attr( $option_name ) . 'rubrics', true ) ) && in_array( $rubric->name, get_post_meta( intval( get_the_ID() ), esc_attr( $option_name ) . 'rubrics', true ) ) ) : ?>
-						checked
-						<?php endif; ?>
-					/>
 					<?php echo esc_html( $rubric->label ); ?>
 				</label>
+				<input
+					type="checkbox"
+					name="<?php echo esc_attr( $option_name ) . 'rubrics[]'; ?>"
+					value="<?php echo esc_attr( $rubric->name ); ?>"
+					<?php if ( \UBC\CTLT\WPVote\WP_Vote_Settings::is_object_rubric_valid( $rubric->name, intval( get_the_ID() ), 'COMMENT' === $object_title ) ) : ?>
```

Comment:
Can we use this, please: https://developer.wordpress.org/reference/functions/checked/